### DISCUSSION TITLE
Internet Explorer (IE) 11: Error was not caught ReferenceError: 'Proxy' is undefined

### DISCUSSION THREAD

**Discussion by robksawyer** [vercel/next.js#19269] [Help]
# Bug report

NextJS 10.0.0 isn't working in IE 11. I'm getting the following error.

```
Error was not caught ReferenceError: 'Proxy' is undefined
```

See https://github.com/marmelab/react-admin/issues/3881 for more.

## To Reproduce

1. Open your NextJS site in IE 11.

## Expected behavior

It should show the browser is unsupported or show the site.

## Screenshots

<img width="1449" alt="Screen Shot 2020-11-18 at 11 23 56 PM" src="https://user-images.githubusercontent.com/179419/99544052-3652f780-29f7-11eb-8bc0-5cb0851350dc.png">

## System information

- OS: Windows
- Browser (if applies): IE 11
- Version of Next.js: 10.0.0
- Version of Node.js: v15.2.0 (npm v7.0.10)
- Deployment: next start

**Comment by robksawyer** [comment]
Update: Still happening in NextJS 10.0.2.

**Comment by Timer** [comment]
Next.js does not use the Proxy API. This is likely your user code. Please provide a fully reproducible demo.

**Comment by bernhardw** [comment]
Are you using Framer Motion? I have the same problem.
See https://github.com/framer/motion/issues/364
